examples/data/cleansed/metadata.yml

```diff
@@ -0,0 +1,61 @@
+scada:
+  time: time
+  id: id
+  WTUR_W: WTUR_W
+  WMET_HorWdSpd: WMET_HorWdSpd
+  WMET_HorWdDir: WMET_HorWdDir
+  status: status
```

Comment:
From the -25 tag list, it seems like WTUR.TurSt is the right tag name for turbine status

Reply:
Updated.